portal.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: redis-operator
+  title: Redis Operator
+  description: "Redis Operator creates/configures/manages high availability redis with sentinel automatic failover atop Kubernetes."
+  links:
+    - url: https://Github.com/powerhome/redis-operator
+      title: Github
+      icon: public
+  annotations:
+    backstage.io/techdocs-ref: dir:.
+    github.com/project-slug: powerhome/redis-operator
+spec:
+  owner: forever-people
+  domain: technology-foundations-and-tools
+  lifecycle: experimental
```

Comment:
Is this correct?

Reply:
🤔 

Addressed by: https://github.com/powerhome/redis-operator/pull/23/commits/cd7e76ca30de86a24e94cbf056a962e88dedbf4f